### PR TITLE
Update Shelly Gen1 minimum firmware versions

### PR DESCRIPTION
**Pull request by thecode**
## Proposed change

Update Shelly Gen1 minimum firmware versions

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/107593
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by thecode** [comment]
Note to reviewers: Due to a device which does not meet the minimum firmware requirement we lowered the minimum firmware requirement for Gen1 devices, this is why we need to update the docs for `current` branch together with the `core` PR

**Review by bieniu — DISMISSED** [review]

**Comment by kblin** [comment]
I'm not sure the gen2 version requirement is good either. I certainly have a plus plug S v1 that stubbornly claims that 0.12.99 is the latest firmware available.

File: source/_integrations/shelly.markdown [hunk 1]
```diff
@@ -310,8 +310,15 @@ Please check from the device Web UI that the configured server is reachable.
 
 ## Known issues and limitations
 
-- Only supports firmware 1.11 and later for generation 1 devices
+- Only supports firmware 1.9 and later for generation 1 devices
 - Only supports firmware 1.0 and later for generation 2 devices
+- The following generation 1 devices only supported from firmware 1.11 and later (due to incompatible API):
```
Comment (c0ffeeca7) [inline source/_integrations/shelly.markdown]:
```suggestion
- The following generation 1 devices only support firmware 1.11 and later (due to incompatible API):
```

**Comment by c0ffeeca7** [comment]
> I'm not sure the gen2 version requirement is good either. I certainly have a plus plug S v1 that stubbornly claims that 0.12.99 is the latest firmware available.

@thecode: does this affect this PR, or would this be a topic for another PR?

**Comment by kblin** [comment]
I think you're good to go. My plug did manage to grab a new firmware after a factory reset and works as described in this PR now.
Sorry for the noise.

**Review by bdraco — APPROVED** [review]

**Comment by thecode** [comment]
Thanks @bdraco

**Comment by thecode** [comment]
and @c0ffeeca7 for the typo fix 👍

**Pull request merged**